doc/source/cluster/kubernetes/user-guides/kuberay-gcs-ft.md

```diff
@@ -6,6 +6,13 @@ By default, the GCS lacks fault tolerance as it stores all data in-memory, and a
 To make the GCS fault tolerant, you must have a high-availability Redis.
 This way, in the event of a GCS restart, it retrieves all the data from the Redis instance and resumes its regular functions.
 
+```{admonition} Fate-sharing without GCS fault tolerance
+Without GCS fault tolerance, the Ray cluster, the GCS process, and the Ray head Pod are fate-sharing.
+If the GCS process dies, the Ray head Pod dies as well after `RAY_gcs_rpc_server_reconnect_timeout_s` seconds.
+If the Ray head Pod is restarted according to the Pod's `restartPolicy`, worker Pods attempt to reconnect to the new head Pod.
+However, the worker Pods are terminated by the new head Pod; without GCS fault tolerance enabled, the cluster state is lost, and the worker Pods are perceived as "unknown workers" by the new head Pod.
```

Comment:
It would be good to add some kind of a concluding sentence with a recommendation for the user, even if it's brief or obvious. (e.g. "To avoid this, ...") Right now it kind of ends on a hopeless note.

Reply:
Updated https://github.com/ray-project/ray/pull/39787/commits/83be4eee1a939f472badf56c2ef6ac5784d45207

Reply:
This is much better!